semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -37,6 +39,73 @@
 from opentelemetry.semconv.templating.markdown.options import MarkdownOptions
 
 
+class VisualDiffer:
```

Comment:
I'm not sure how I feel about this new class being "together" with the rest of the actual functionality of the generator. I wonder if it wouldn't be better if it was in a separated file for ex.

Reply:
I guess just a file on its own, TableDiffHelper or something :D